lib/player.js

```diff
@@ -3846,6 +3848,9 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
             shaka.util.Error.Code.TEXT_ONLY_WEBVTT_SRC_EQUALS,
             mimeType);
       }
+      if (forced) {
+        kind = 'forced';
```

Comment:
Is this some Apple feature to force the cues to be displayed?  Wouldn't it be better to just set the `mode` to `showing`?  This seams platform-specific and doesn't appear in the HTML spec.

Reply:
This is the way Safari works. I have seen it testing with https://devimages.apple.com.edgekey.net/streaming/examples/bipbop_16x9/bipbop_16x9_variant.m3u8 and other private streams.

Reply:
The I suggest at least a comment.  I'd also be concerned with browser support.  This is supposed to be an enum value, so some browsers may throw if we set to an invalid value.  Maybe you should set the `kind` after we set the original value and do it inside a try-catch.

Reply:
This change is only used in src = and since you have to create a track element a try-catch is not effective in this case.

Tried an unknown kind in Chrome and it works fine.

Reply:
Safari: https://github.com/WebKit/webkit/blob/ae1c6da14bb6bf86cd92d9bfb3d5670043b3c00e/Source/WebCore/html/track/TextTrack.cpp#L134-L150

Chrome: https://bugs.chromium.org/p/chromium/issues/detail?id=271185

More info: https://github.com/whatwg/html/iss

Reply:
Here's the correct html spec issue link: https://github.com/whatwg/html/issues/4472

Reply:
I would suggest replacing the comment with a link to the HTML spec issue.

Reply:
Done!